### PR TITLE
DCOS-9159: Disable tasks' restart button while its service is deploying

### PR DESCRIPTION
**Pull request by jfurrow**
This PR disables the task restart button when the service is deploying.

Here's the tooltip that appears:

### CODE REVIEW

**Comment by mlunoe** [comment]
When I am not hovering it looks like this: 

Is that intended? It looks kinda weird to me

**Comment by jfurrow** [comment]
@mlunoe I suspect your version of CNVS is out of date, this is what the button's disabled static state should look like:

File: plugins/services/src/js/containers/tasks/TasksView.js [hunk 1]
```diff
@@ -143,42 +145,63 @@ class TasksView extends mixin(SaveStateMixin) {
...
     // Using Button's native "disabled" prop prevents onMouseLeave from
     // being correctly called, preventing correct dismissal of the Tooltip.
     //
-    // So we manually disable the button.
+    // So we overwrite the click handlers manually.
+    let handleRestartClick = function () {};
     let handleStopClick = function () {};
 
+    if (!isDeploying) {
+      handleRestartClick = this.handleStopClick.bind(this, 'restart');
```
Comment (mlunoe) [inline plugins/services/src/js/containers/tasks/TasksView.js]: This might also need a rename:

Comment (jfurrow) [inline plugins/services/src/js/containers/tasks/TasksView.js]: Thanks! Fixed!

**Comment by jfurrow** [comment]
Rebased with integration test fixes 🎉

**Comment by jfurrow** [comment]
Rebased with integration test fixes 🎉 v2

**Comment by philipnrmn** [comment]
Nice! Thanks for you patience on the merge @jfurrow :)

**Pull request merged**